src/pages/namespace/Monitoring/Logs/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Loader2, ScrollText } from "lucide-react";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from "~/design/Tooltip";
+
+import { ButtonBar } from "~/design/ButtonBar";
+import CopyButton from "~/design/CopyButton";
+import ScrollContainer from "./Scrollcontainer";
+import { formatLogTime } from "~/util/helpers";
+import { useNamespacelogs } from "~/api/namespaces/query/logs";
+import { useTranslation } from "react-i18next";
+
+const LogsPanel = () => {
+  const { t } = useTranslation();
+  const { data } = useNamespacelogs();
+
+  const copyValue =
+    data?.results.map((x) => `${formatLogTime(x.t)} ${x.msg}`).join("\n") ?? "";
+
+  const resultCount = data?.results.length ?? 0;
+
+  return (
+    <>
+      <div className="mb-5 flex flex-col gap-5 sm:flex-row">
+        <h3 className="flex grow items-center gap-x-2 font-medium">
+          <ScrollText className="h-5" />
+          {t("pages.monitoring.logs.title")}
+        </h3>
+        <ButtonBar>
+          <TooltipProvider>
+            <Tooltip>
+              <TooltipTrigger asChild>
+                <div className="flex grow">
+                  <CopyButton
+                    value={copyValue}
+                    buttonProps={{
+                      variant: "outline",
+                      size: "sm",
+                      className: "grow",
+                    }}
+                  />
+                </div>
+              </TooltipTrigger>
+              <TooltipContent>
+                {t("pages.monitoring.logs.tooltips.copy")}
+              </TooltipContent>
+            </Tooltip>
+          </TooltipProvider>
+        </ButtonBar>
+      </div>
+      <ScrollContainer />
+      <div className="flex items-center justify-center pt-2 text-sm text-gray-11">
+        <Loader2 className="h-3 animate-spin" />
```

Comment:
I don't have an obvious solution for this and it is ok to leave it as is for now. But intuitively, the spinning circle feels like something is wrong (as if there is content that should finish loading, but it's stuck). 

I am thinking that something like three pulsating dots (like a typing notification in a chat) could make sense here. I don't know if something like this exists in Lucide, but maybe the [ping](https://tailwindcss.com/docs/animation#ping) or pulse notification with a suitable icon would work?

Reply:
I know what you mean. I tried the "someone is writing" kind of animation at first when I created the instances page and I agree that it feels like it is stuck in a loading state even if it is just a long-running stream with no data coming in.

The ping is nice! I've implemented it for both log streaming pages

![CleanShot 2023-08-14 at 15 25 59](https://github.com/direktiv/direktiv-ui/assets/121789579/41dc3106-a0d6-429a-86bf-0dbaa0273a45) (e8d928e84367333baa3db9f13a34c502e2a7bfae)